README.md

```diff
@@ -40,13 +47,15 @@ Feel free to contribute!
 * [news-crawl](https://github.com/commoncrawl/news-crawl): Cralwer for news feeds based on StromCrawler that prouduces WARC files.
 * [Plowshare](https://github.com/mcrapet/plowshare): Command-line tool to manage file-sharing site
 * [Rclone](https://github.com/ncw/rclone): A command line program to sync files and directories to and from various cloud storage providers
+* [rsync](https://rsync.samba.org/): An open source utility that provides fast incremental file transfer
+* [Suck-It](https://github.com/skallwar/suckit): Recursively visit and download a website's content to your disk (multi-threaded)
 * [wget](https://savannah.gnu.org/git/?group=wget): Utility for non-interactive download of files from the Web.
+* [wget2](https://gitlab.com/gnuwget/wget2): Successor of GNU Wget, works multi-threaded
 * [wpull](https://github.com/ArchiveTeam/wpull): Wget-compatible web downloader and crawler
 * [you-get](https://github.com/soimort/you-get): Dumb downloader that scrapes the web
 * [Youtube-DL](https://github.com/rg3/youtube-dl): A command-line program to download videos from YouTube and a few hundred more sites
 
 ### Application-specific
```

Comment:
No "back to top" here, is that intentional?

Reply:
That's intentional. The "back to top" links were only added to 2nd level headings, like Awesome-Selfhosted has them.